src/VisualStudio/Core/Def/Implementation/EditAndContinue/VsENCRebuildableProjectImpl.cs

```diff
@@ -783,8 +783,23 @@ public int GetENCBuildState(ShellInterop.ENC_BUILD_STATE[] pENCBuildState)
                     }
                     else if (_encService.EditSession != null)
                     {
-                        _projectBeingEmitted = _vsProject.VisualStudioWorkspace.CurrentSolution.GetProject(_vsProject.Id);
-                        _lastEditSessionSummary = GetProjectAnalysisSummary(_projectBeingEmitted);
+                        // Fetch the latest snapshot of the project and get an analysis summary for any changes 
+                        // made since the break mode was entered.
+                        var currentProject = _vsProject.VisualStudioWorkspace.CurrentSolution.GetProject(_vsProject.Id);
```

Comment:
Here we're asserting that it's non-null.  I'm not sure if we have a reason.
http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/EditAndContinue/EditAndContinueDiagnosticUpdateSource.cs,82